client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -53,8 +53,9 @@ const badge = (
 
 const getTitanFeatures = () => {
 	return [
-		translate( 'Inbox, calendars, and contacts' ),
+		translate( 'Send and receive from your custom domain' ),
 		translate( '30GB storage' ),
+		translate( 'Email, calendars, and contacts' ),
```

Comment:
Should we call out that users can access their email from WordPress.com as a feature? We may need to wait for the embedded inbox to ship before we promote that feature.

Reply:
We can add it for now without highlighting it maybe.

Reply:
I would vote for waiting to avoid confusing users.

Reply:
Sounds good